### ISSUE TITLE
5Ghz Wi-fi Speed

### ISSUE THREAD

**Issue by markymmp** [syscl/XPS9350-macOS#196]
Hi i think i've asked before but my 5Ghz wi-fi speed is very slow and appears to be interfered with by the bluetooth radio.. any advice please.?

**Comment by TheRedMudder** [comment]
What wireless card are you using? I am guessing it is DW 1560, so did you patch AirPortBrcm4360.kext? Or are you using DW1830? Or something else?

**Comment by markymmp** [comment]
I've upgraded the card to a DW1830..

**Comment by TheRedMudder** [comment]
I think I had the same exact issue on my old Optiplex Hackintosh. 

Does the issue start when you wake the computer from sleep? I am not sure where I read it, but genuine DW1830 from DELL have a slightly different firmware than the ones on eBay. Some issue with the firmware causes slow downs after the computer is put to sleep on macOS. The solution is to create a script that turns the card off right before the device sleeps and turns it back on right before you wake up from sleep. I believe there is a guide on tonymac somewhere on how to do this. Good luck!

**Comment by markymmp** [comment]
Yup it does it from coming out of standby so yup bit of a pain and is an Ebay purchase.. Many thanks for the response.. Will have a look over on TM..
